src/wodan/wodan.mli

```diff
@@ -15,31 +15,58 @@
 (*                                                                              *)
 (********************************************************************************)
 
-val max_dirty : int
+(** The Wodan module allows access to Wodan filesystems
+
+    The entry point is {!Wodan.Make}, which returns a
+    {!Wodan.S} module supporting filesystem operations *)
 
 exception BadMagic
+(** Raised when trying to open a superblock and it doesn't have the magic tag for Wodan filesystems *)
 
 exception BadVersion
+(** Raised when trying to open a superblock and it doesn't have a supported version for Wodan filesystems *)
 
 exception BadFlags
+(** Raised when trying to open a superblock and it sets must-support flags that this version doesn't support *)
+
+exception BadCRC of int64
+(** Raised when a block doesn't have the expected CRC
 
-exception BadCRC of Location.t
+    The exception carries the logical offset of the block. *)
 
 exception BadParams
+(** Raised when {!SUPERBLOCK_PARAMS} passed to the Make functor don't match actual settings saved in the superblock *)
```

Comment:
Those look like long lines; apparently we haven't enabled `parse-docstrings` in `.ocamlformat` yet; we should probably do that.

Reply:
Sure. Added